jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/frames/PushPromiseFrame.java

```diff
@@ -26,6 +26,16 @@
     private final int promisedStreamId;
     private final MetaData metaData;
 
+    public PushPromiseFrame(int streamId, MetaData metaData)
+    {
+        this(streamId, 0, metaData);
+    }
+
+    public PushPromiseFrame(int promisedStreamId, PushPromiseFrame frame)
+    {
+        this(frame.getStreamId(), promisedStreamId, frame.getMetaData());
+    }
```

Comment:
I would remove this constructor, as it is only used by the implementation. I would prefer to not expose it to applications.

Reply:
sure.